src/alf/util/useColorModeTheme.ts

```diff
@@ -46,6 +46,13 @@ function updateDocument(theme: ThemeName) {
     html.className = html.className.replace(/(theme)--\w+/g, '')
 
     html.classList.add(`theme--${theme}`)
+
+    // set color to 'theme-color' meta tag
+    const themeColor = window
+      .getComputedStyle(html)
+      .getPropertyValue('--background')
```

Comment:
Would it make more sense if we got the background color from the theme directly here, instead of getting it from the document's styles?